sysdig/config.ini

```diff
@@ -0,0 +1,28 @@
+[insightfinder]
+license_key = a333bd39662ad1edf64e99e97e4b776109827234
```

Comment:
please add comments in this config to make each item clear and remove your testing values

Reply:
Done.